lib/code_ownership/cli.rb

```diff
@@ -1,4 +1,5 @@
 # typed: true
+# frozen_string_literal: true
```

Comment:
rubocop didn't like the unfrozen `EXECUTABLE` below, which I've fixed via magic comment, although it's a riskier fix.